lib/inject_method.rb

```diff
@@ -0,0 +1,11 @@
+class Array
+
+  def inject_method(result, &block)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used.